compiler/rawdata2hdf5/rawdata2hdf5

```diff
@@ -105,6 +106,7 @@ def create_hdf5(data_list, output_path):
 
 
 def main():
+    warnings.warn("[rawdata2hdf5] section will be deprecated. Please use [one-create-quant-dataset] section.");
```

Comment:
Q) what is `[rawdata2hdf5] section` ?

Reply:
@seanshpark 

> Q) what is [rawdata2hdf5] section ?

Simply, it means the tool of `rawdata2hdf5`. 😅
I was thinking about what would be better for this warning message, 
and I used a message referring to this. --- > [reference](https://github.com/Samsung/ONE/blob/c91b55370e28993569059233f744a0085d0dbbea/compiler/one-cmds/onelib/CfgRunner.py#L66)


Is it better to use the word `tool` than the word `section`?

Reply:
@jinevening , any comments?

Reply:
> and I used a message referring to this. 

do you understand the word `section` from the reference?

Reply:
@Seunghui98  `rawdata2hdf5` can be called in the command line, so `section` seems not appropriate. Could you update the message as below?

```suggestion
    warnings.warn("rawdata2hdf5 will be deprecated. Please use one-create-quant-dataset.");
```

Reply:
@jinevening 
Thank you for your comment :)

Reply:
FYI `section` came from https://en.wikipedia.org/wiki/INI_file#Sections